app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,36 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if isinstance(lhs, str):
+            return int(lhs) + rhs
+        return lhs + rhs
+
+    @staticmethod
+    def format_ordinal(number_to_format, determiner=None):
+        number_to_format_str = str(number_to_format)
```

Comment:
A cleaner and quicker (timeit) version you could use is 

suffix = {1: "st", 2: "nd", 3: "rd"}.get(
number_to_format % 10 * (number_to_format % 100 not in ["11", "12", "13"]), "th")

Reply:
this is real head-scratcher as it failed unit tests on 11, 12, 13(spitting out 11st, 12nd, 13rd) and got linted in strange manner with come extra comas, I opted for Andrew's solution in the end. Thanks👍